Source/WebCore/platform/graphics/mac/controls/ControlMac.h

```diff
@@ -42,6 +42,13 @@ class ControlMac : public PlatformControl {
     ControlMac(ControlPart&, ControlFactoryMac&);
 
 protected:
+    static constexpr float m_baseFontSize = 11.0f;
+    static constexpr float m_baseArrowHeight = 4.0f;
+    static constexpr float m_baseArrowWidth = 5.0f;
+    static constexpr float m_baseSpaceBetweenArrows = 2.0f;
+    static constexpr int m_arrowPaddingBefore = 6;
+    static constexpr int m_arrowPaddingAfter = 6;
+
```

Comment:
Can these just be in `MenuListButtonMac.mm`?

Reply:
They are moved to `MenuListButtonMac::draw()`